adr/0011-storing-diversity-data.md

```diff
@@ -0,0 +1,113 @@
+# 11. Storing Diversity Data
+
+Date: 2019-01-14
+
+## Status
+
+Accepted
+
+## Context
+
+This service will need to store sensitive diversity data for each candidate - gender, ethnicity, etc - which, although of little interest to attackers, would be potentially impactful to the candidate if someone was to gain unauthorised access to it. Therefore we need to take extra care to minimise the risk of this happening.
+
+## Findings
+
+The risks and threats we considered:
+
+1. Non-malicious but still unauthorised access
+for instance, a genuine provider or support user just clicking around out of interest, or to check up on a family member, etc
```

Comment:
The formatting makes this read a bit oddly here — github concatenates the lines in the `<ol>`. Full stop?